cmd/globaldns.go

```diff
@@ -734,6 +734,14 @@ func globalDNSUpdate(ctx *cli.Context) error {
 			return err
 		}
 		update["multiClusterAppId"] = resource.ID
+		if len(entry.ProjectIDs) > 0 {
+			input := &managementClient.UpdateGlobalDNSTargetsInput{
+				ProjectIDs: entry.ProjectIDs,
+			}
+			if err := c.ManagementClient.GlobalDNS.ActionRemoveProjects(entry, input); err != nil {
+				return err
```

Comment:
What if the update fails, we should rollback to the previous state and set back the Projects?